examples/components/MNIST/aggregatemodelweights/aggregatemodelweights.yaml

```diff
@@ -34,4 +34,6 @@ command: >-
   [--input_silo_2 ${{inputs.input_silo_2}}]
   [--input_silo_3 ${{inputs.input_silo_3}}]
 
-environment: azureml:AzureML-sklearn-0.24-ubuntu18.04-py37-cpu@latest
+environment: 
+  conda_file: ./conda.yaml
+  image: mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04
```

Comment:
did you use a cpu image intentionally? if running torch on gpu you will need to use a gpu / cuda image

ex: openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04

Reply:
I used the CPU image intentionally since we are creating CPU compute clusters. Please let me know if we want to move everything on the GPU. Closing it for now. Happy to push these changes in another PR if required.